frb_dart/lib/src/basic.dart

```diff
@@ -26,18 +27,20 @@ abstract class FlutterRustBridgeBase<T extends FlutterRustBridgeWireBase> {
   void _sanityCheckSingleton() {
     if (_instances.contains(runtimeType)) {
       throw Exception(
-          'Subclasses of `FlutterRustBridgeBase` should be singletons - there should not be two instances (runtimeType=$runtimeType)');
+        'Subclasses of `FlutterRustBridgeBase` should be singletons - '
+        'there should not be two instances (runtimeType=$runtimeType)',
+      );
     }
     _instances.add(runtimeType);
   }
 
   void _setUpRustToDartComm() {
-    inner.store_dart_post_cobject(NativeApi.postCObject.cast());
+    inner.storeDartPostCObject();
   }
 
   /// Execute a normal ffi call. Usually called by generated code instead of manually called.
   @protected
-  Future<S> executeNormal<S>(FlutterRustBridgeTask<S> task) {
+  Future<S> executeNormal<S>(FlutterRustBridgeTask<S> task) async {
```

Comment:
Maybe this "async" need not to be used, since we are using a `Completer` instead of `await`s